client/packages/common/src/ui/layout/tables/components/Cells/CheckboxCell/EnabledCheckboxCell.tsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import { CellProps } from '../../../columns';
+import { RecordWithId } from '@common/types';
+import { CheckboxCell } from './CheckboxCell';
+
+export const EnabledCheckboxCell = <T extends RecordWithId>({
```

Comment:
because, if you disable the entire row, you cannot re-enable it 😃

Reply:
How were you able to "disable" an entire row, though? Cos if I turn off all the "Count" checkboxes, the "OK" button is disabled so can't save that state.

Not particularly relevant to your fix, was just wondering how it gets to that "all disabled" state in the first place.